django_mongodb/base.py

```diff
@@ -54,7 +54,25 @@ class DatabaseWrapper(BaseDatabaseWrapper):
         "TimeField": "date",
         "UUIDField": "string",
     }
+    # Django uses these operators to generate SQL queries before it generates
+    # MQL queries.
```

Comment:
Can you elaborate on this? Why does SQL get generated if we're running a mongodb query?

Reply:
It has to do with how this library hooks in to the query generation process. See also:

https://github.com/mongodb-labs/django-mongodb/blob/3c3ad0eb45a2572453aa228ddafac96391dc1ab8/django_mongodb/query.py#L23-L32